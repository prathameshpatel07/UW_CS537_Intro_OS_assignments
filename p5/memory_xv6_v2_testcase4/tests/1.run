cd v2_testcase4; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_1; cd ..
