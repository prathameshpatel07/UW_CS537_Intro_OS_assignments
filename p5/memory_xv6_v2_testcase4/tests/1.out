spawn make CPUS=1 -f Makefile.test qemu-nox
qemu-system-i386 -nographic -drive file=fs.img,index=1,media=disk,format=raw -drive file=xv6.img,index=0,media=disk,format=raw -smp 1 -m 512 
xv6...
cpu0: starting 0
sb: size 1000 nblocks 941 ninodes 200 nlog 30 logstart 2 inodestart 32 bmap start 58
init: starting sh
$ test_1
Frames: DFFF PIDs: 1
Frames: DFC7 PIDs: 3
Frames: DFC6 PIDs: 3
Frames: DFC5 PIDs: 3
Frames: DFC4 PIDs: 3
Frames: DFC3 PIDs: 3
Frames: DFC2 PIDs: 3
Frames: DFC1 PIDs: 3
Frames: DFC0 PIDs: 3
Frames: DFBE PIDs: 2
Frames: DFBC PIDs: 3
Frames: DF7A PIDs: 1
Frames: DF78 PIDs: 1
Frames: DF77 PIDs: 1
$ QEMU: Terminated
